content/_changelogs/11.2.0.md

```diff
@@ -0,0 +1,29 @@
+## 11.2.0
+
+_Released 11/22/2022_
+
+**Features:**
+
+- Re-implemented the 'Run All Specs' for End to End Testing for running multiple
+  tests sequentially in open mode. This feature had been previously removed in
+  Cypress 10.0. Enable 'Run All Specs' with the new experimental flag,
+  `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature.
+  Addresses [#24168](https://github.com/cypress-io/cypress/issues/24168). tests
+  sequentially in Open Mode for End to End testing. Addresses
+  [#24168](https://github.com/cypress-io/cypress/issues/24168) and
+  [this discussion](https://github.com/cypress-io/cypress/discussions/21628)
+- Added messaging in CI to direct people to set up recording to the Cypress
```

Comment:
I found this hard to parse, how about:

> Added messaging in CI, after test failures, which directs people to record test results to the Cypress Cloud if they are not already set up to do so. Addresses
  [#24000](https://github.com/cypress-io/cypress/issues/24000)

Reply:
I tweaked it a little but I think it's similar to what you have here (and better than what I had previously)